io/swig/io/vview.py

```diff
@@ -918,22 +890,36 @@ def init_cf_sources(mu):
         transforms[instance] = []
         offsets[instance] = []
         actors[instance] = []
+        times_of_birth[instance] = io.instances()[instance_name].\
+                                   attrs['time_of_birth']
+
+        if instance > 0:
```

Comment:
Argh.. actors[instance] should not exist as actors is splitted in dynamic_actors and static_actors, I miss this one when I rebased.
And yes, it is instance >= 0 for dynamic objects...